Change config from JSON to a JS object

andreabettich opened the pull request.

czosel commented on backend/.gitignore:
```diff
@@ -1,3 +1,4 @@
 coverage
 config.json
```
shall we drop this line now?

andreabettich replied:
makes sense :smile:

czosel commented on backend/config.example.js:
```diff
@@ -0,0 +1,127 @@
+    port: 6379,
+    options: {}
+  },
+  services: [
```
Suggestion for another PR: I never understood why `services` is an array with objects containing a `type` property. Wouldn't refactoring to something like
```
services: {
  redmine: {...},
  vault: {...}
}
```
be much simpler?

andreabettich replied:
for sure it would be simpler, than searching over the array for the service

andreabettich commented:
can we merge this and I'll change the services in the configuration in another PR

andreabettich commented:
I changed the services and removed the Adfinis Sygroup information in the example configuration.
Added some infos/comments into the `README.md` and into the `config.example.js`

czosel commented on backend/src/services.js:
```diff
@@ -8,29 +8,18 @@ import vaultCustom from './vault/vault-custom'
+router.use('/proxy/symon', SymonProxy.createProxy(symon))
+
+router.use('/vault', vaultCustom(vault))
+router.use('/proxy/vault', VaultProxy.createProxy(vault))
```
This is so much easier to read :+1:
